plone/namedfile/tests/test_storable.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# This file was borrowed from z3c.blobfile and is licensed under the terms of
+# the ZPL.
+
+##############################################################################
+#
+# Copyright (c) 2008 Zope Foundation and Contributors.
+# All Rights Reserved.
+#
+# This software is subject to the provisions of the Zope Public License,
+# Version 2.1 (ZPL).  A copy of the ZPL should accompany this distribution.
+# THIS SOFTWARE IS PROVIDED "AS IS" AND ANY AND ALL EXPRESS OR IMPLIED
+# WARRANTIES ARE DISCLAIMED, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF TITLE, MERCHANTABILITY, AGAINST INFRINGEMENT, AND FITNESS
+# FOR A PARTICULAR PURPOSE.
+#
+##############################################################################
+
+from OFS.Image import Pdata
+from plone.namedfile.file import FileChunk
+from plone.namedfile.file import NamedBlobImage
+from plone.namedfile.tests.base import getFile
+from plone.namedfile.tests.base import NamedFileLayer
+
+import unittest
+
+
+class TestStorable(unittest.TestCase):
+
+    layer = NamedFileLayer
+
+    def setUp(self):
+        pass
+
+    def test_pdata_storable(self):
+        pdata = Pdata(getFile('image.gif').read())
+        fi = NamedBlobImage(pdata, filename=u'image.gif')
+        self.assertEquals(303, fi.getSize())
+
+    def test_str_storable(self):
+        fi = NamedBlobImage(getFile('image.gif').read(), filename=u'image.gif')
+        self.assertEquals(303, fi.getSize())
```

Comment:
@vangheem code analysis does not like ``assertEquals`` and prefers ``assertEqual``. This is because its deprecated https://docs.python.org/2/library/unittest.html#deprecated-aliases